doc/version_history.rst

```diff
@@ -6,6 +6,25 @@
 Version History
 ###############
 
+v1.13.2
+-------
+
+* `Model`: when enabling the model, call the alarm callback for all alarms, even those in nominal state.
+  Most alarms will usually be in nominal state.
+* `WatcherCsc`:
+
+    * When enabling the CSC: print alarm events for all alarms, even those in nominal state.
+      Most alarms will usually be in nominal state.
+    * ``showAlarms`` command: print an alarm event for all events, even those in nominal state.
+
+Requires:
+
+* ts_utils 1.1
+* ts_salobj 7.1
+* ts_idl 2
+* IDL files for ``Watcher``, ``ATDome``, ``ESS``, ``MTMount``, ``ScriptQueue``, and ``Test``, plus any additional SAL components you wish to watch.
+  These may be generated using ``make_idl_files.py`` built with ts_xml 13 and ts_sal 7.
```

Comment:
I wonder if the `ts_xml` version you mention here is still sufficient. Did you test that? Perhaps this needs to be upped to 16 by now?

Reply:
Good catch. I did add the `notification` event, and any rule that uses it will require ts_xml 16. We don't have any of those yet. I'll think about how to say that.